adam/ontology/phase1_ontology.py

```diff
@@ -242,15 +256,114 @@ def subtype(sub: OntologyNode, _super: OntologyNode) -> None:
 subtype(DESTINATION, SEMANTIC_ROLE)
 
 # Hierarchical structure of objects
-_HEAD_SCHEMA = ObjectStructuralSchema(_HEAD)
 _TORSO_SCHEMA = ObjectStructuralSchema(_TORSO)
-_ARM_SCHEMA = ObjectStructuralSchema(_ARM)
+# _ARM_SCHEMA = ObjectStructuralSchema(_ARM)
 _LEG_SCHEMA = ObjectStructuralSchema(_LEG)
 _CHAIRBACK_SCHEMA = ObjectStructuralSchema(_CHAIR_BACK)
 _CHAIR_SEAT_SCHEMA = ObjectStructuralSchema(_CHAIR_SEAT)
 _TABLETOP_SCHEMA = ObjectStructuralSchema(_TABLETOP)
 _TAIL_SCHEMA = ObjectStructuralSchema(_TAIL)
 _WING_SCHEMA = ObjectStructuralSchema(_WING)
+_EYE_SCHEMA = ObjectStructuralSchema(_EYE)
```

Comment:
Let's pull person back down to the simple HEAD + ARM + LEGS stick figure, except put hands on the end of the ARMS. The hands don't need any internal structure yet.